_rules/meta-viewport-b4f0c3.md

```diff
@@ -36,15 +36,20 @@ For each test target, the `content` attribute, whose value is mapped to a list o
 
 ## Assumptions
 
+If any of the following is true, this rule can file while the [1.4.4 Resize text][sc144] and [1.4.10 Reflow][sc1410] can still be satisfied:
+
 - The [page][] has [visible][] [content][].
 - There is no other [mechanism](https://www.w3.org/TR/WCAG21/#dfn-mechanism) available to resize the text content.
+- The [content][] is not by default rendered in a way that fits in an area of 320 by 256 [CSS pixels][], and needs to reflow to do so.
 
 ## Accessibility Support
 
 Desktop browsers ignore the viewport `meta` element, and most modern mobile browsers either ignore it by default or have an accessibility option which will allow zooming. This rule is not relevant for desktop browsers, nor for most modern mobile browsers. Only users with older mobile browsers can experience issues tested by this rule.
 
 ## Background
 
+This rule is designed specificially for [1.4.4 Resize text][sc144], which requires that text can be resized up to 200%. Because text that can not be resized up to 200% will can not fit in an area of 320 by 256 [CSS pixels][], this rule maps to [1.4.10 Reflow][sc1410] as well. All passed examples in this rule satisfy both success criteria.
```

Comment:
```suggestion
This rule is designed specificially for [1.4.4 Resize text][sc144], which requires that text can be resized up to 200%. Because text that can not be resized up to 200% can not fit in an area of 320 by 256 [CSS pixels][], this rule maps to [1.4.10 Reflow][sc1410] as well. All passed examples in this rule satisfy both success criteria.
```